direct/io-hbase-bindings/src/main/java/cz/o2/proxima/direct/hbase/HBaseLogReader.java

```diff
@@ -119,32 +136,29 @@ public void observe(
   private void flushPartitions(
       List<Partition> partitions,
       List<AttributeDescriptor<?>> attributes,
+      TerminationContext terminationContext,
       BatchLogObserver observer)
       throws IOException {
 
+    outer:
```

Comment:
👎

Reply:
Simplyfying code, nothing wrong with this.